<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilenameWidget</class>
 <widget class="QWidget" name="FilenameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="filenameEdit"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="filenameButtonLayout"/>
   </item>
   <item>
    <widget class="QCheckBox" name="destinationCheck">
     <property name="text">
      <string>on save, move file into sub directory:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="destinationButtonLayout"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
